<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QLabel" name="playerLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>83</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Player</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="playerText">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>325</width>
         <height>33</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="playerSelect">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="privacy">
      <attribute name="title">
       <string>Privacy</string>
      </attribute>
      <widget class="QCheckBox" name="clickTracking">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Click tracking is used primarily for history features. If this feature is turned off, watch history will be stored locally instead of with your YouTube account.</string>
       </property>
       <property name="text">
        <string>Enable click tracking</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
